client/post-editor/editor-drawer/index.jsx

```diff
@@ -12,7 +12,7 @@ import Accordion from 'components/accordion';
 import AccordionSection from 'components/accordion/section';
 import Gridicon from 'components/gridicon';
 import CategoriesTagsAccordion from 'post-editor/editor-categories-tags/accordion';
-import EditorSharingAccordion from 'post-editor/editor-sharing/accordion';
+import EditorSharingAccordion from 'async-component!post-editor/editor-sharing/accordion';
```

Comment:
>how does this work with tests?

Probably not very well, and I'm sure same goes for base `<AsyncLoad />`. Will need to think about this some more.